### PR TITLE
kn-apply task

### PR DESCRIPTION
**Pull request by Aneesh-M-Bhat**

### CODE REVIEW

**Comment by Aneesh-M-Bhat** [comment]
Might have to adjust the default values for the test, PTAL

**Review by vdemeester — APPROVED** [review]
/lgtm

File: hack/install-osp.sh [hunk 1]
```diff
@@ -113,9 +138,35 @@ EOF
 rollout_status "openshift-pipelines" "tekton-pipelines-controller"
 rollout_status "openshift-pipelines" "tekton-pipelines-webhook"
 
+install_serverless
+
+# wait until serverless operator is created
+echo "Waiting for OpenShift Serverless Operator to be created..."
```
Comment (vdemeester) [inline hack/install-osp.sh:142]: We may need to update the timeout to 3 or 4 minutes maybe.

Comment (Aneesh-M-Bhat) [reply hack/install-osp.sh:142]: yep updated with 4m

**Pull request merged**